assets/datatables/crops_dt_row.gd

```diff
@@ -1,8 +1,10 @@
 class_name CropConfigRow extends DatatableRow
 
+@export var crop_id: String
 @export var name: String
```

Comment:
Might be better to separate the seed conversion and crop details DTs so you can do quick lookup on either.

Reply:
Will look at in a future PR indexing by crop instead of seed